### DISCUSSION TITLE
PCL 1.12 is out, will it be updated soon in the cask?

### DISCUSSION THREAD

**Discussion by kunaltyagi** [Homebrew/discussions#1898] [Everyday usage]
1.12 doesn't depend on VTK8 anymore. I don't know how to edit recipes, or I'd have created a PR.

A nominal change is here: https://github.com/kunaltyagi/homebrew-core/tree/patch-1

**Comment by carlocab** [comment]
Thanks! Those changes are mostly right. Have there been no other changes to the dependencies other than in VTK?

**Reply by carlocab** [reply]
I opened Homebrew/homebrew-core#82029 to update the formula.

**Reply by kunaltyagi** [reply]
Nice. 

> Have there been no other changes to the dependencies other than in VTK?

One that I know of: There has been a better support for qHull (https://github.com/PointCloudLibrary/pcl/pull/4540/files), but depending on modules being built, it might not have an impact
